tools/hooks/requirements_changed.rb

```diff
@@ -7,7 +7,8 @@
   "package.json" => "yarn",
   "bower.json" => "bower install",
   "Gemfile" => "bundle install",
-  "Berksfile" => "berks install"
+  "Berksfile" => "berks install",
+  "schema.rb" => "rake db:migrate"
```

Comment:
Do these "file patterns" accept paths? We could do `"dashboard/db/schema.rb"` just in case we ever get another `schema.rb` like you suggested.